projects/envoy/Dockerfile

```diff
@@ -33,3 +33,4 @@ RUN apt-get update && apt-get -y install  \
 RUN git clone https://github.com/envoyproxy/envoy.git
 WORKDIR $SRC/envoy/
 COPY build.sh $SRC/
+COPY WORKSPACE $SRC/envoy/
```

Comment:
Thanks, I'll try this, though not sure if it plays nicely with the rest of the oss-fuzz containers.

Reply:
After trying to use the proposed change (adding a user instead of replacing the workspace file), got the following error:
```
INFO:__main__:Keeping existing build artifacts as-is (if any).
INFO:__main__:Running: docker run --rm --privileged --shm-size=2g --platform linux/amd64 -i -e FUZZING_ENGINE=libfuzzer -e SANITIZER=address -e ARCHITECTURE=x86_64 -e PROJECT_NAME=envoy -e HELPER=True -e FUZZING_LANGUAGE=c++ -v /<path>/git/oss-fuzz/build/out/envoy/:/out -v /<path>/git/oss-fuzz/build/work/envoy:/work -t gcr.io/oss-fuzz/envoy.
---------------------------------------------------------------
Compiling libFuzzer to /usr/lib/libFuzzingEngine.a... cp: cannot create regular file '/usr/lib/libFuzzingEngine.a': Permission denied
```

So using non-root will require additional changes (IIRC the output is also in a directory owned by root).
As Envoy's WORKSPACE file doesn't change often, I think overriding it is an adequate solution for now.

Reply:
cool - was just a suggestion really - if you are happy that you are not exposing anything by using root in the container then i think it should be ok - its kinda annoying that rules_python enforce this, but good that someone does in a way